modules/olmv1-troubleshooting-rbac-errors-with-preflight-check.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * extensions/ce/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="olmv1-troubleshooting-rbac-errors-with-preflight-check_{context}"]
+= Preflight permissions check for cluster extensions (Technology Preview)
```

Comment:
Do we normally put "(Technology Preview)" in the headings of actual modules? I don't think I've seen this done very often. The only place I've seen that done is in headings for release notes. I see this is done in a module right next to this, so it's fine to leave it for now at least, but I'd consider in the future whether this is a practice we should be doing or not.

Reply:
Interesting. I'll add a topic to the Monday meeting.

I feel like this is something I did with Alex for a while in the Operator Framework. I'll look back to previous releases after merge freeze to double check my memory.